src/commands/classified/pipeline.rs

```diff
@@ -31,15 +31,15 @@ pub(crate) async fn run_pipeline(
             (_, Some(ClassifiedCommand::Error(err))) => return Err(err.clone().into()),
 
             (Some(ClassifiedCommand::Internal(left)), _) => {
-                run_internal_command(left, ctx, input, Text::from(line)).await?
```

Comment:
These weren't problematic now, but my concern was that if we ever did await on a command to complete, it may prevent us from running all commands in a pipeline simultaneously.